facebook/Makefile

```diff
@@ -124,7 +125,7 @@ params.weekly-weights.json: $(TODAY)
 		debug false \
 		produce_individual_raceeth true \
 		end_date $(LAST_SATURDAY) \
-		input <(find $(QUALTRICS) -maxdepth 1 -newer $(LAST_SUNDAY) -not -newer $(shell date -d "$(LAST_SATURDAY) +2 day" +"%Y-%m-%d") -type f -name "*.csv" | sort | grep $${PAT}  | tr '\n' ',' | sed 's_$(QUALTRICS)/__g;s/,$$//' ) \
+		input <(find $(QUALTRICS) -maxdepth 1 -newer $< -type f -name "*.csv" | sort | grep $${PAT}  | tr '\n' ',' | sed 's_$(QUALTRICS)/__g;s/,$$//' ) \
```

Comment:
hmm this will only capture the files that are downloaded the day the pipeline runs, which only contain survey responses from the last 7 days. depending on the day of the week, you might have as little as 1 day of output available for the time span from start_date $(LAST_SUNDAY) to end_date $(LAST_SATURDAY). is that the intended behavior?

Reply:
The last 14 days of data should be loaded from the archive, so I don't think this will be a problem.